internal/server/agentProxy.go

```diff
@@ -213,8 +212,8 @@ func (proxy *AgentProxy) defaultHandler(w http.ResponseWriter, r *http.Request)
 	}
 
 	log.Info("Sending response")
-	n, _ := io.Copy(w, res.Body)
-	LogExit(log, r, res.StatusCode, int(n))
+	w.WriteHeader(http.StatusOK)
```

Comment:
```suggestion
	w.WriteHeader(res.StatusCode)
```
this should be the status code from the remote response, not OK.

Reply:
Done! Thanks!